core/standalone/build.gradle

```diff
@@ -37,30 +38,41 @@ scoverage {
     scoverageScalaVersion.set("${gradle.scala.scoverageScalaVersion}")
 }
 
+task downloadSwagger(type: Download) {
+    src "https://github.com/swagger-api/swagger-ui/archive/v${gradle.swagger.version}.tar.gz"
+    dest file("$buildDir/tmp/swagger/swagger-ui.tar")
+    overwrite false
+}
+
+
 task copySwagger(type: Copy) {
-    def version = "3.6.0"
     mkdir("$buildDir/tmp/swagger")
     def destFile = file("$buildDir/tmp/swagger/swagger-ui.tar")
     def uiDir = file("$buildDir/tmp/swagger/swagger-ui")
-    if (!destFile.exists()) {
-        ant.get(src: "https://github.com/swagger-api/swagger-ui/archive/v${version}.tar.gz", dest: destFile)
-    }
     from(tarTree(resources.gzip(destFile))){
-        include("swagger-ui-${version}/dist/**")
+        include("swagger-ui-${gradle.swagger.version}/dist/**")
     }
     into(uiDir)
     includeEmptyDirs = false
-    project.ext.swaggerUiDir = file("$buildDir/tmp/swagger/swagger-ui/swagger-ui-${version}/dist")
+    project.ext.swaggerUiDir = file("$buildDir/tmp/swagger/swagger-ui/swagger-ui-${gradle.swagger.version}/dist")
 }
 
+copySwagger.dependsOn downloadSwagger
```

Comment:
I would move that into the task definition

Reply:
It should be done with 80959f9832cce44ad64de3bee2a391f00a907e9e